app/models/concerns/updatable.rb

```diff
@@ -76,6 +76,7 @@ def update_parent_objects
       next unless parent_object
       sets << ', ' + parent_object.admin_set.key
       split_sets = sets.split(',').uniq.reject(&:blank?)
+      admin_set = editable_admin_set(row['admin_set'], oid, index) unless row['admin_set'].blank?
```

Comment:
`editable_admin_set` can return `false`, in which case I think the logic should skip the row?   But it could also be `nil`.